priv/templates/jsonapi/view.ex

```diff
@@ -0,0 +1,19 @@
+defmodule <%= module %>View do
+  use <%= base %>.Web, :view
+
+  use JSONAPI.PhoenixView
+
+  def type, do: "<%= singular %>"
+
+  def attributes(model) do
+    Map.take(model, [:id, :created_at])
```

Comment:
Should probably also generate this similar to https://github.com/phoenixframework/phoenix/blob/master/priv/templates/model/model.ex